packages/utils/src/abi_decoder.ts

```diff
@@ -37,10 +58,10 @@ export class AbiDecoder {
     public tryToDecodeLogOrNoop<ArgsType extends DecodedLogArgs>(log: LogEntry): LogWithDecodedArgs<ArgsType> | RawLog {
         const methodId = log.topics[0];
         const numIndexedArgs = log.topics.length - 1;
-        if (_.isUndefined(this._methodIds[methodId]) || _.isUndefined(this._methodIds[methodId][numIndexedArgs])) {
+        if (_.isUndefined(this._eventIds[methodId]) || _.isUndefined(this._eventIds[methodId][numIndexedArgs])) {
             return log;
         }
-        const event = this._methodIds[methodId][numIndexedArgs];
+        const event = this._eventIds[methodId][numIndexedArgs];
         const ethersInterface = new ethers.utils.Interface([event]);
```

Comment:
Should we remove the `ethers` dependency and use `AbiDecoder` instead? Maybe a TODO for another PR?

Reply:
Let's make this a TODO for another PR. I'm blocked by getting this merged.

Reply:
Yeah there's some additional work needed to replace interfaces. We have a task in the protocol backlog to finish replacing ethers throughout the monorepo.